src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -110,15 +111,18 @@ public AppealController(AppealService appealService) {
         @ApiResponse(responseCode = "500", description = "Internal server error")
     })
     @GetMapping(value = "/{company-id}/appeals", produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<Appeal> getAppealByPenaltyReference(@PathVariable("company-id") final String companyId,
+    public ResponseEntity<List<Appeal>> getAppealByPenaltyReference(@PathVariable("company-id") final String companyId,
                                                               @RequestParam(value="penaltyReference") final String penaltyReference) {
 
         LOGGER.info("GET /{}/appeals?penaltyReference={}", companyId, penaltyReference);
```

Comment:
I know this is not part of this ticket, but could you update the logging statement to include `/companies` in the url?